app/components/Poll/index.js

```diff
@@ -78,14 +78,11 @@ class Poll extends React.Component<Props, State> {
             <Icon name="stats" />
             <span className={styles.pollHeader}>{title}</span>
           </Link>
-          <Tooltip
-            content="Avstemningen er anonym."
-            style={{ marginLeft: '-180px' }}
-          >
+          <Tooltip content="Avstemningen er anonym." renderToThe={'left'}>
```

Comment:
You can write `renderToThe="left"` instead

Reply:
Thanks, I didn't know that!